README.md

```diff
@@ -268,6 +268,14 @@ Thus, these should all be set on the JVM command line using `-D`.
   Else, the JCA will get the implementation from another registered provider (usually stock JCE).
   Using JCE's impelmentation is generally recommended unless using ACCP as a standalone provider
   Callers can choose to register ACCP's implementation at runtime with a call to `AmazonCorrettoCryptoProvider.registerEcParams()`
+* `com.amazon.corretto.crypto.provider.registerSecureRandom`
+  Takes in `true` or `false` (defaults to `false` in FIPS mode, `true` in non-FIPS).
+  If `true`, then ACCP will register a SecureRandom implementation (`LibCryptoRng`) backed by AWS-LC
+  Else, ACCP will not register a SecureRandom implementation, meaning that the JCA will source SecureRandom instances from another registered provider.
+  LibCryptoRng is very fast during steady state operation in all cases. In FIPS mode, however, AWS-LC-FIPS's CPU jitter-based entropy source incurs a ~10ms initialization cost for every new thread.
+  This means that there is a slight "pause" before randomness production in highly threaded environments.
+  Because, in extreme cases, this could prevent an availability risk, we do not register LibCryptoRng by default in cases where this initialization cost is incurred (i.e. FIPS mode).
```

Comment:
>this could prevent an availability risk

I think "prevent" should be "present" here.

Reply:
hm, i agree. pretty much the opposite of what we're trying to say... thanks. will fix.